test/test_files/dml_node/set/set_read.test

```diff
@@ -63,3 +63,40 @@ Alice2021-06-30
 Bob1950-05-14
 Carol2000-01-01
 Dan
+
+-CASE SetMultipleReadTest
+-STATEMENT CREATE NODE TABLE play(ID INT64, name STRING, score INT64, PRIMARY KEY(ID));
+---- ok
+-STATEMENT CREATE (a:play {ID: 0, name: 'AA', score: 8});
+---- ok
+-STATEMENT MATCH (a:organisation:play) SET a.name = 'X', a.score = 42 RETURN a.ID, a.name, a.score;
+---- 4
+0|X|42
+1|X|42
+4|X|42
+6|X|42
+-STATEMENT MATCH (a:organisation:play) RETURN a.ID, a.name, a.score;
+---- 4
+0|X|42
+1|X|42
+4|X|42
+6|X|42
+
+-CASE SetByDictReadTest
+-STATEMENT CREATE NODE TABLE play(ID INT64, name STRING, score INT64, PRIMARY KEY(ID));
+---- ok
+-STATEMENT CREATE (a:play {ID: 0, name: 'AA', score: 8});
+---- ok
+-STATEMENT MATCH (a:organisation:play) SET a = { name: 'X', score: 42 } RETURN a.ID, a.name, a.score;
```

Comment:
I want more testing for this feature which should include

- Set edge property.
- Data type coverage. Let's set/update all properties appears in `tinysnb` dataset `person` table. 
- Mismatched data type, e.g. try to set a int property with string value
- Incorrect variable name, e.g. `MATCH (a:organisation:play) SET x={}`
- Set with empty dictionary.
- Set when value is NULL.

Ideally these cases should have already been covered. Since you are just rewriting, this feature should automatically inherits existing implementation and no change is needed. I'm requesting these tests to ensure this.